single_cell/workflows/qc/tasks.py

```diff
@@ -0,0 +1,112 @@
+import os
+import pypeliner.commandline 
+import glob
+import gzip
+import shutil
+from single_cell.utils import helpers
+import pandas as pd
+
+def merge_mafs(mafs, merged_maf, id_colname=False):
+    assert isinstance(mafs, dict)
+    preppedmafs = []
+    for samplegroup, maf in mafs.items():
+        maftemp = pd.read_csv(maf, sep="\t", skiprows=1)
+        maftemp = maftemp.astype({"Chromosome": "str","PHENO": "str","SOMATIC": "str","PUBMED": "str"})
+        if id_colname:
+            maftemp["id"] = [samplegroup] * len(maftemp)
+        preppedmafs.append(maftemp)
+        
+    output = pd.concat(preppedmafs).reset_index()
+    output = output[output.t_alt_count > 2]
+    output.to_csv(merged_maf, sep="\t", index=False, header=True)
+
+def merge_snvs(snv_files, merged_snv, id_colname=False):
+    
+    assert isinstance(snv_files, dict)
+    preppednsvs = []
+    for samplegroup, snv_file in snv_files.items():
+        svtemp = pd.read_csv(snv_file).astype({"chrom": "str"})
+        if id_colname:
+            svtemp["id"] = [samplegroup] *len(svtemp)
+        svtemp = svtemp[svtemp.alt_counts > 1]
+        preppednsvs.append(svtemp)
+    output = pd.concat(preppednsvs).reset_index()
+    output.to_csv(merged_snv, sep="\t", index=False, header=True)
+
+
+def filter_snvs_for_high_impact(snv, filtsnv):
+    
+    cmd = ["mergesnvs.R", snv, filtsnv]    
+    pypeliner.commandline.execute(*cmd, docker_image="rocker/tidyverse")
+
+def filter_maf_for_high_impact(maf, filtmaf):
+    cmd = ["mergemafs.R", maf, filtmaf]
+    pypeliner.commandline.execute(*cmd, docker_image="rocker/tidyverse")
+
+def _get_indelvcfs(dir):
+    indel_vcfs = []
+    indel_vcfs += sorted(glob.glob(dir + "/results/variants/" + "*_strelka_indel.vcf.gz" ))
+    indel_vcfs += sorted(glob.glob(dir + "/results/" + "*_strelka_indel.vcf.gz" ))
+    indel_vcfs += sorted(glob.glob(dir + "/results/variant_calling/*/" + "*indel.vcf.gz" ))
+    return indel_vcfs
+
+#dummie outputs for testing
+def dummie(files):
+    for file in files:
+        df = pd.DataFrame([1], [1])
+        df.to_csv(file)
+    return
+
+def get_snv_all_csvs(dir):
+    output = {}
+    dir_len = len(dir.split("/"))
+    filenames =  sorted(glob.glob(dir + "/*/*/*snv*all*csv", recursive=True))
+    for filename in filenames:
+
+        name_len = len(filename.split("/"))
+        samplelabel = filename.split("/")[dir_len : name_len - 1]
+        output[tuple(samplelabel)] = filename
+    return output
+
+def gunzip_file(infile, outfile):
```

Comment:
see if this exists in single_cell.utils.helpers. 

if it doesnt, move it there.